src/main/java/com/cloudbees/jenkins/support/actions/SupportObjectAction.java

```diff
@@ -67,6 +68,8 @@ public String getIconFileName() {
         return "/plugin/support-core/images/24x24/support.png";
     }
 
+    protected abstract String getBundleNameQualifier();
```

Comment:
I think this would break if any other components is implementing an Action. Maybe providing a default value would be a better option.